### PR TITLE
SDK-Core version bump to 0.0.5

### PR DESCRIPTION
**Pull request by adamcronin42**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -13384,13 +13384,18 @@
                 "node": ">=14"
             }
         },
+        "packages/node/node_modules/@backtrace-labs/sdk-core": {
```
Comment (konraddysput) [inline package-lock.json:13387]: I think you didn't bump everything

Comment (adamcronin42) [reply package-lock.json:13387]: Or did you mean, bump the version but skip the actual node release for now?

Comment (adamcronin42) [reply package-lock.json]: I went ahead and included node

Comment (konraddysput) [reply package-lock.json]: Yeah this is exactly what I meant. Do not release a new node library yet

**Review by konraddysput — APPROVED** [review]

**Pull request merged**